src/Microsoft.ML.Data/Evaluators/EvaluatorMetrics.cs

```diff
@@ -0,0 +1,93 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Microsoft.ML.Data.Evaluators
+{
+    public class EvaluatorMetrics
```

Comment:
>EvaluatorMetrics [](start = 17, length = 16)

maybe call this one BinaryClassificationMetrics, and the other CalibratedBinaryClassificationMetrics.